shopelectro/management/commands/price.py

```diff
@@ -1,3 +1,4 @@
+from collections import defaultdict
```

Comment:
As I remember in this project we stick to such sort-rules:
1. First the expression "import ..." and then the expression "from ... import ..."
2. We divide imports into three groups: built-in, third and local modules